packages/wrangler/src/index.tsx

```diff
@@ -1911,8 +1911,14 @@ export async function main(argv: string[]): Promise<void> {
     if (e instanceof CommandLineArgsError) {
       wrangler.showHelp("error");
       console.error(""); // Just adds a bit of space
+      console.error(e.message);
+    } else {
+      console.error(e.message);
+      console.error(""); // Just adds a bit of space
+      console.error(
```

Comment:
should this maybe be a `.warn()` or green text if we can swing it? 

We could also add the 🐛

Reply:
I'm not that keen on emoji in my console 😄 
Let me see if I can colour it without it messing up the CI...

Reply:
I have added the colour codes but they don't show up in Github differ